src/EmployerAan/SFA.DAS.EmployerAan/Models/GetMemberProfileWithPreferencesModel.cs

```diff
@@ -34,7 +36,19 @@ public class GetMemberProfileWithPreferencesModel
         IsRegionalChair = memberProfileWithPreferences.IsRegionalChair;
         Profiles = memberProfileWithPreferences.Profiles;
         Preferences = memberProfileWithPreferences.Preferences;
-        Apprenticeship.ActiveApprenticesCount = employerMemberSummaryResult.ActiveCount;
-        Apprenticeship.Sectors = employerMemberSummaryResult.Sectors;
+        if (memberProfileWithPreferences.UserType == MemberUserType.Employer && employerMemberSummaryResult != null)
+        {
+            Apprenticeship.ActiveApprenticesCount = employerMemberSummaryResult.ActiveCount;
+            Apprenticeship.Sectors = employerMemberSummaryResult.Sectors;
+        }
+        if (myApprenticeship != null && myApprenticeship.TrainingCourse != null)
```

Comment:
Since this is not embedded in else, which is needs to be, would you rather care to add the UserType == Apprentice here

Reply:
Done